### ISSUE TITLE
its work into EAC game ?

### ISSUE THREAD

**Issue by skullcheat** [SurgeGotTappedAgain/Pink-Eye#1]
Hello nice share its work for EAC game ?

**Comment by MmCopyVirtualMemory** [comment]
Nah, this is Battleye specific. It relies on MmGetSystemRoutineAddress being used.

https://github.com/SurgeGotTappedAgain/Pink-Eye/blob/a51b25f56fcec717998aa5ed86aef83dc8c95214/BEEmulator/Source%20Files/Main.cpp#L81
